static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -42,10 +48,27 @@ const formFields: Field[] = [
 ];
 
 function SentryFunctionDetails(props: Props) {
+  const api = useApi();
   const form = useRef(new FormModel());
   const {orgId, functionSlug} = props.params;
+  const {sentryFunction} = props;
   const method = functionSlug ? 'PUT' : 'POST';
-  const endpoint = `/organizations/${orgId}/functions/`;
+  let endpoint = `/organizations/${orgId}/functions/`;
+  if (functionSlug) {
+    endpoint += `${functionSlug}/`;
+  }
+  const defaultCode = sentryFunction
+    ? sentryFunction.code
+    : `exports.yourFunction = (req, res) => {
+    let message = req.query.message || req.body.message || 'Hello World!';
+    console.log('Query: ' + req.query);
+    console.log('Body: ' + req.body);
+    res.status(200).send(message);
+  };`;
+
+  useEffect(() => {
+    form.current.setValue('code', defaultCode);
```

Comment:
@vpandiarajan20 So the constructor for the form takes an argument for `initialData`, you can just pass in the data that way.